scripts/distribution/builder.Dockerfile

```diff
@@ -75,9 +75,14 @@ ENV DISTRIBUTION_CLIENT=true
 ENV CONCORDIUM_NODE_COLLECTOR_URL=https://dashboard.${environment}/nodes/post
 ENV CONCORDIUM_NODE_COLLECTOR_GRPC_HOST=http://localhost:10000
 
-RUN apt-get update && apt-get install -y curl netbase ca-certificates supervisor nginx libnuma1 libtinfo6 libpq-dev liblmdb-dev jq
-RUN curl -L https://getenvoy.io/install.sh | bash -s -- -b /usr/local/bin
-RUN getenvoy use 1.18.3
+RUN apt-get update && \
+    apt-get install -y curl netbase ca-certificates supervisor nginx libnuma1 libtinfo6 libpq-dev liblmdb-dev jq apt-transport-https gnupg2 curl lsb-release && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
I'm not clear what the purpose of this line is, and why it's here specifically.

Reply:
The `rm` line? It's [recommended best practice](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run) to reduce the size of the image by removing the repo cache.

Reply:
Seems reasonable. Should it also be added at the end of the next run step?

Reply:
Yes!